### PR TITLE
belindas_closet_4_142_github_documentation

### PR DESCRIPTION
**Pull request by brandondombrowsky**
Completion of "main" ReadMe. Relates to User Story 142 within belindas-closet-android.

### CODE REVIEW

**Review by taylorpapke — DISMISSED** [review]
Very well structured. It is easy to read and find the information that I am looking for quickly.

There is a typo on line 20 where it says "Benlinda's"

**Comment by brandondombrowsky** [comment]
Fixed the type. Good eye @taylorpapke!

**Comment by brandondombrowsky** [comment]
As soon as this is approved I can copy/paste/edit and then polish the nsc-events ReadMe, for which I hope to have a PR ready for review by EOD Monday.

**Review by bcko — APPROVED** [review]

**Pull request merged**
